views/index.erb

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style> 
```

Comment:
Best practice would be to extract a style sheet for your CSS to keep CSS and HTML code separate. But I'm just trying to give you constructive feedback. You've already gone way beyond me...

Reply:
I need to research how to do this! But it sounds like it would make the code DRYer